bugbot/rules/component.py

```diff
@@ -89,6 +89,36 @@ def get_bugs(self, date="today", bug_ids=[]):
         # Classify those bugs
         bugs = get_bug_ids_classification("component", bug_ids)
 
+        # Collect bugs classified as Fenix::General
+        fenix_general_bug_ids = [
+            bug_id
+            for bug_id, bug_data in bugs.items()
+            if bug_data.get("class") == "Fenix::General"
+        ]
+
+        # Reclassify the Fenix::General bugs using the fenixcomponent model
+        if fenix_general_bug_ids:
+            fenix_general_classification = get_bug_ids_classification(
+                "fenixcomponent", fenix_general_bug_ids
+            )
```

Comment:
With this, we use the model to predict the Fenix component for any bug that the component model classifies as `Fenix::General` even if they have very low confidence. Such bugs are usually filtered out of the training set for the Fenix-only model. My concern here is that we are exposing the model to types of bugs that were not trained on.

Reply:
I agree that there could be cases where the Fenix-specific model could encounter bug types that it has not been trained on. What do you think of increasing the component confidence threshold for the Fenix-specific model? If it does not meet the threshold it would just default to the general component model classification (i.e. `Fenix::General`).

Reply:
Yes, we should use the threshold that gave us good results on the testing dataset.

Reply:
I've introduced a new confidence threshold for Fenix-specific components in `rules.json` in db0adb583d5e336d7dd41cf945d37ab1c606dea6. It's currently set to 0.6 (from [here](https://docs.google.com/spreadsheets/d/1ocDf-sTuk8AFqUDBYtYMUtR7uwJ0Nk5_8k7EO9DsXCI/edit?usp=sharing)) but can be changed accordingly.

Reply:
I see that "No CT" gives us higher recall without losing on the precision compared to "0.6". What do you think?